vcd/resource_vcd_ip_space_test.go

```diff
@@ -258,6 +258,7 @@ resource "vcd_ip_space" "space1" {
 `
 
 const testAccVcdIpSpacePublicStep5DS = testAccVcdIpSpacePublicStep5 + `
+# skip-binary-test: Data Source test
 data "vcd_ip_space" "space1" {
   name = "{{.TestName}}"
 }
```

Comment:
I will check - the reason I am adding this is because:

We used this resource and data source approach in a single file for a long time, but then Terraform stopped supporting it at some time (even with `depends_on`) - all these tests started failing. It needed a lot of work to split all the test work. This happened a few years ago

Reply:
Added `depends_on` and ran tests in acceptance and binary test modes to be sure that it works